src/Umbraco.Web.UI.Client/src/views/propertyeditors/blocklist/umbBlockListPropertyEditor.component.js

```diff
@@ -220,8 +220,12 @@
                 });
             }
 
-            // TODO: Why is there a '/' prefixed? that means this will never work with virtual directories
-            block.view = (block.config.view ? "/" + block.config.view : getDefaultViewForBlock(block));
+            if(block.config.view) {
+                block.config.view = umbRequestHelper.convertVirtualToAbsolutePath(block.config.view);
```

Comment:
We should not be pushing back this to config. If the config is wrong - well then it should not work and a fix should be applied to make sure it doesn't contain bad data, instead of simply "fixing things" here if the data is wrong.